prboom2/src/SDL/i_sound.c

```diff
@@ -1305,14 +1308,15 @@ char music_player_order[NUM_MUS_PLAYERS][200] =
   PLAYER_DUMB,
   PLAYER_FLUIDSYNTH,
   PLAYER_OPL2,
+  PLAYER_ALSA,
```

Comment:
Please stand in line. The portmidi backend is here for decades, it is known working and does so out-of-the-box. Also is the new backend, with no working default configuration yet and currently tested by only one user - its developer. Please add it last, thanks!

Reply:
Indeed. This is an experimental backend, it shouldn't be preferred over a true and tested one like Portmidi. I should have put more thought into the ordering. I'll swap it into place.